### PR TITLE
Sample Task function and parser

### PR DESCRIPTION
**Pull request by e0426051**
Probably don't merge thanks

### CODE REVIEW

File: src/main/java/scrumptious/TaskRunner.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
+package scrumptious;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Scanner;
+
+public class TaskRunner {
+    public static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void main(String[] args) {
```
Comment (gmit22) [inline src/main/java/scrumptious/TaskRunner.java]: I think here the main implementation is for parser to be used for your testing?

Comment (keke101) [reply src/main/java/scrumptious/TaskRunner.java]: Yeap, Haoxiang, remove this file then commit again to the same branch, then we will merge this pull request

Comment (e0426051) [reply src/main/java/scrumptious/TaskRunner.java]: ok I removed the file.
Is the scrumptious package name ok though

Comment (keke101) [reply src/main/java/scrumptious/TaskRunner.java]: I think it is ok, we can leave it as it is now and we will settle the package names later on

Comment (e0426051) [reply src/main/java/scrumptious/TaskRunner.java]: ok. I merge? Or someone merge after checking or sth?

**Comment by keke101** [comment]
Looks good to me, can merge imo.

File: src/main/java/scrumptious/TaskRunner.java (hunk 1 above)
Comment (keke101) [reply src/main/java/scrumptious/TaskRunner.java]: I think wait for the next person to review then merge.

**Pull request merged**